e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const defaultAboutLink = 'https://about.mattermost.com/default-about/';
+
+describe('System Console', () => {
+    before(() => {
+        // # Update config to defaults
+        cy.apiUpdateConfig({
+            SupportSettings: {
+                aboutLink: defaultAboutLink,
+            },
+        });
+    });
```

Comment:
Different from what I projected in the ticket but could you please setup the before and after similar to [other PR](https://github.com/mattermost/mattermost-webapp/pull/4330/files#diff-e8e5668a9849acec0e1a98afb29a0883R11-R29).  We normally don't revert state on `after` hook but we're making exception here.